LondonBritishLibrary/orient/BLorient11601.xml

```diff
@@ -233,6 +225,54 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 </handNote>
                             </handDesc>
+                            <additions>
+                                <list>
+                                    <item xml:id="e2">
+                                        <locus target="#7r"></locus>
+                                        <q xml:lang="gez">አመ፭ለጰግሜን፡ አሞጽ።</q>
+                                    </item>
+                                    <item xml:id="e3">
+                                        <locus target="#13v"></locus>
+                                        <q xml:lang="gez">አመ፡ ፳ወ፪፡ ለነሐሴ፡ ሚክያስ።</q>
+                                    </item>
+                                    <item xml:id="e4">
+                                        <locus target="#18v"></locus>
+                                        <q xml:lang="gez">አመ፡ ፳ወ፩ለጥቅምት፡ ኢዩኤል።</q>
+                                    </item>
+                                    <item xml:id="e5">
+                                        <locus target="#21va"></locus>
+                                        <q xml:lang="gez">አመ፡ ፳ወ፫ለኅዳር፡ አብድዩ።</q>
+                                    </item>
+                                    <item xml:id="e6">
+                                        <locus target="#22v"></locus>
+                                        <q xml:lang="gez">አመ፡ ፳ወ፭ለመስከረም፡ ዮናስ።</q>
+                                    </item>
+                                    <item xml:id="e7">
+                                        <locus target="#24v"></locus>
+                                        <q xml:lang="gez">አመ፡ ፭ለታኅሣሥ፡ ናሆም።</q>
+                                    </item>
+                                    <item xml:id="e8">
+                                        <locus target="#26v"></locus>
+                                        <q xml:lang="gez">አመ፡ ፫ለኅዳር፡ እንባቆም።</q>
+                                    </item>
+                                    <item xml:id="e9">
+                                        <locus target="#29r"></locus>
+                                        <q xml:lang="gez">አመ፡ ፬ለሐምሌ፡ ሶፎንያስ።</q>
+                                    </item>
+                                    <item xml:id="e10">
+                                        <locus target="#31v"></locus>
+                                        <q xml:lang="gez">አመ፡ ፳ለታኅሣሥ፡ ሐጌ።</q>
+                                    </item>
+                                    <item xml:id="e11">
+                                        <locus target="#33r"></locus>
+                                        <q xml:lang="gez">አመ፡ ፲ወ፭፡ ለየካቲት፡ ዘካርያስ።</q>
+                                    </item>
+                                    <item xml:id="e12">
+                                        <locus from="1" to="45"/>
+                                        <desc type="findingAid">Various explanatory notes in the margins.</desc>
```

Comment:
these are probably glosses or similar, not `findingAid `- findingAid are titles/weekdays etc added for better navigation